src/App.js

```diff
@@ -12,10 +12,25 @@ export default function App() {
 
   // Write code here.
   //
+  useEffect(() => {
+    console.log("Data Type: " + dataType)
+    getData()
+  },[dataType])
+
+  async function getData() {
+    if (dataType !== '') {
+      const response = await fetch('https://swapi.dev/api/' + dataType + '/')
```

Comment:
I'd recommend string interpolation here so you can insert variables into a string instead of concatenating, e.g.

```js
const response = await fetch(`https://swapi.dev/api/${dataType}/`)
```